retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2021 Google Inc. All Rights Reserved.
+﻿// Copyright 2022 Google Inc. All Rights Reserved.
```

Comment:
Please revert, we don't update the copyright year when updating existing samples.

Reply:
done